package.json

```diff
@@ -5,29 +5,44 @@
   "description": "Foreman isn't really a node module, these are just dependencies needed to build the webpack bundle. 'dependencies' are the asset libraries in use and 'devDependencies' are used for the build process.",
   "private": true,
   "devDependencies": {
+    "babel-cli": "^6.10.1",
     "babel-core": "~6.7.2",
     "babel-eslint": "^6.1.2",
     "babel-loader": "~6.2.4",
+    "babel-plugin-transform-class-properties": "^6.11.5",
     "babel-preset-es2015": "~6.6.0",
+    "babel-preset-react": "^6.5.0",
+    "babel-preset-stage-0": "^6.5.0",
+    "babel-register": "^6.9.0",
     "css-loader": "^0.23.1",
     "es6-promise": "^3.2.1",
     "eslint": "^3.3.1",
+    "eslint-plugin-react": "^6.1.2",
     "expose-loader": "~0.6.0",
     "extract-text-webpack-plugin": "^1.0.1",
     "file-loader": "^0.9.0",
+    "script-loader": "^0.7.0",
     "stats-webpack-plugin": "^0.2.1",
     "style-loader": "^0.13.1",
     "url-loader": "^0.5.7",
     "webpack": "^1.9.11",
     "webpack-dev-server": "^1.9.0"
   },
   "dependencies": {
+    "babel-polyfill": "^6.9.1",
+    "c3": "^0.4.11",
+    "events": "^1.1.1",
+    "flux": "^2.1.1",
     "jquery": "~1.11.0",
     "jquery-flot": "~0.8.3",
     "jquery-ujs": "~1.2.0",
     "jquery.cookie": "~1.4.1",
     "jstz": "~1.0.7",
-    "lodash": "~2.4.1",
+    "lodash": "~4.13",
```

Comment:
Yes, I needed the update a long time ago. After the other PR is merged and I rebase, this will go away. Now I need it.